### DISCUSSION TITLE
docker stability

### DISCUSSION THREAD

**Discussion by 2426852837** [SCADA-LTS/Scada-LTS#2609] [General]
My computer is Win11 and SCADALTS starts normally after executing the docker-compose up command using the files provided by dockerhub. But I go to localhost:9090 and the interface keeps loading without any interface or prompt. It regularly returns localhost without sending any data. I would like to ask if this is because the docker version you provide is not stable enough?

This is my docker-compose.yml

version: '3'
services:
database:
container_name: mysql
image: mysql/mysql-server:5.7
ports:
- "13306:3306"
environment:
- MYSQL_ROOT_PASSWORD=root
- MYSQL_USER=root
- MYSQL_PASSWORD=root
- MYSQL_DATABASE=scadalts
expose: ["3306"]
volumes:
- ./docker/volumes/databases:/home/
scadalts:
image: scadalts/scadalts:latest
ports:
- "9090:8080"
depends_on:
- database
expose: ["8080", "8000"]
links:
- database:database

**Comment by Limraj** [comment]
Hi @2426852837,
after all, you got the answer what needs to be done here: https://github.com/SCADA-LTS/Scada-LTS/issues/2604

Regards,
Kamil Jarmusik

**Reply by 2426852837** [reply]
I followed the steps you sent me to do the check, but I found that part of the time it was accessible through localhost:9090, and it took about five minutes to still have the above problem

**Reply by Limraj** [reply]
If you had followed the steps, you would firstly use the current docker-compose configuration file, and secondly, you would send more information describing the problem so that we know exactly what you mean. You haven't done any of these steps, so why do you say you did any steps?

**Comment by Limraj** [comment]
hi @2426852837,
1. try to use this config file: https://github.com/SCADA-LTS/Scada-LTS/blob/develop/docker-compose.yml
At first, do not change anything, if it works, then you will change to another port, ok?
First you need to start the database:
`docker-compose up database`
When the database starts then:
`docker-compose up scadalts`
[docker-compose-tutorial](https://github.com/SCADA-LTS/Scada-LTS/wiki/Scada-LTS-docker-compose-tutorial)

2. If you see any error anything, send a screenshot.


Regards,
Kamil Jarmusik
